### ISSUE TITLE
I'm dumb and don't get it

### ISSUE THREAD

**Issue by pearcemerritt** [KenjiOhtsuka/kotlin-midi-player-sample#2]
Hey Kenji,
Thanks so much for making this repo after I asked you about how to use your other library `kotlin-midi-player`! I'm embarrassed to say that I'm so new to gradle and kotlin that I still don't really know what to do with the project haha.

I tried running `gradle tasks` but nothing seemed to imply running the code...

I got it working for a second in Android Studio and heard the midi, but then it stopped working and I've been trying at the command line since then.

After running `gradle build` I found a jar but can't seem to figure out how to run it:
```
$ java -jar build/libs/midi_sample-1.0-SNAPSHOT.jar
no main manifest attribute, in build/libs/midi_sample-1.0-SNAPSHOT.jar
```

I don't mean to bug you with n00b questions, but I thought I'd post this question here in case there's actually something and it's not just my lack of experience with this tech :)

**Comment by KenjiOhtsuka** [comment]
Hi @pearcemerritt 

I didn't add manufest.

I executed it from green play mark at the left side of `fun main`. I'm using IntelliJ IDEA.

![image](https://user-images.githubusercontent.com/1572810/62602267-5360d580-b92e-11e9-948d-7e7676b3f57b.png)

**Comment by KenjiOhtsuka** [comment]
Hi @pearcemerritt 

I didn't try yet, but the following article will help you if you want an executable jar.

https://medium.com/@preslavrachev/kotlin-basics-create-executable-kotlin-jars-using-gradle-d17e9a8384b9

**Comment by pearcemerritt** [comment]
Hey thanks, 
Good news: I deleted and re-cloned your git repo and Android Studio started working again and I was able to run again by clicking the green triangle you showed in your picture.

Bad news: I went through that guide, but it ultimately didn't help me figure out how to run things from the command line.  But, that's ok, that more of an academic curiosity. I'll close this issue for now :)
